src/olympia/bandwagon/models.py

```diff
@@ -19,76 +17,27 @@
 from olympia.users.models import UserProfile
 
 
-SPECIAL_SLUGS = amo.COLLECTION_SPECIAL_SLUGS
-
-
-class CollectionQuerySet(BaseQuerySet):
-    def with_has_addon(self, addon_id):
```

Comment:
Are we just removing this because it isn't currently used? I know there are still some issues opened on addons-frontend that might make use of this if we ever get around to addressing them. It's unlikely we will - they are issues that keep becoming stale and then are commented on by an interested user, but I wonder if it's worth keeping this functionality around in case a contributor wants to fix those issues themselves? Just a thought.

Reply:
The whole issue was removing code because it wasn't currently used 😉 

For frontend to be able to do anything with this it would need to be exposed via the API... and if _that_ work was done it'd be trivial to reimplement it there.